specification/cognitiveservices/data-plane/Speech/VideoDubbing/preview/2024-05-20-preview/video-dubbing.json

```diff
@@ -0,0 +1,849 @@
+            "schema": {
+              "$ref": "#/definitions/Translation"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/Translation"
+            },
+            "headers": {
+              "Operation-Id": {
+                "type": "string",
+                "description": "ID of the operation."
+              },
+              "Operation-Location": {
+                "description": "The location of the operation job. Use this URL to monitor operation status.",
+                "type": "string",
+                "format": "uri"
+              },
+              "retry-after": {
+                "description": "Indicate the minimum number of seconds the client should wait before polling (GETing) the status monitor URL again for an update.",
+                "type": "string",
+                "format": "uri"
+              }
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a project": {
+            "$ref": "./examples/create_content_editing.json"
+          }
+        }
+      }
+    },
+    "/translations/{id}/edit-operations": {
+      "get": {
+        "tags": [
+          "EditOperations"
+        ],
+        "description": "Gets edit operations list info.",
+        "operationId": "EditOperations_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/PathResourceId"
+          },
+          {
+            "$ref": "#/parameters/QueryApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/PaginatedTranslations"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Operation": {
+            "$ref": "./examples/get_edit_operations.json"
+          }
+        }
+      }
+    },
+    "/translations/{id}/edit-operations/{operation-id}": {
+      "get": {
+        "tags": [
+          "EditOperation"
+        ],
+        "description": "Gets edit operation info.",
+        "operationId": "EditOperations_Get",
```

Comment:
should be "EditOperation_Get". You can find such error by copy swager to https://editor.swagger.io/